presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Kafka.java

```diff
@@ -69,7 +71,8 @@ private DockerContainer createKafka()
                 .withEnv("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "1")
                 .withEnv("KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS", "0")
                 .withStartupCheckStrategy(new IsRunningStartupCheckStrategy())
-                .waitingForAll(forSelectedPorts(9092), forLogMessage(".*started \\(kafka.server.KafkaServer\\).*", 1));
+                .waitingForAll(forSelectedPorts(9092), forLogMessage(".*started \\(kafka.server.KafkaServer\\).*", 1))
+                .withStartupTimeout(Duration.ofMinutes(5));
```

Comment:
Is it safe timeout?  Why not say 10 minutes? Or 1 minute?

Does the timeout including pulling images?

Reply:
Nope, that's not accounted for.